### PR TITLE
Docker update

### PR DESCRIPTION
**Pull request by GAS85**
`docker-run` cleanup
`Dockerfile` update - remove custom CMD, unneded commands
`nginx.conf` convert to template
`README.md` rephrasing

### Description
This change removes nginx container customization and use default nginx ecosystem, such as [Templating](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119) and external scripts execution.
Enable custom port support if needed to remap it.

### Motivation and Context
No need to copy and replace `CMD` parts of nginx container, ensure simplicity to run and security by only needed access rights (remove `chmod -R a+rw`). Use templating to apply Variables, such as `BASE_URL` and `NGINX_PORT`.

### How Has This Been Tested?
Container was build and executed with various parameters.

### My PR contains... 

- [X] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [X] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [X] are not breaking changes.

### Documentation
- [X] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by char0n** [comment]
Hi @GAS85,

Thanks for the changes! Looking into them now

File: docker-run.sh [hunk 1]
```diff
@@ -64,8 +59,6 @@ if [[ "${URL_OAS3_GENERATOR}" ]]; then
   fi
 fi
 
-exec nginx -g 'daemon off;'
```
Comment (char0n) [inline docker-run.sh:67]: TECHNICAL NOTE EXPLAINING THE CHANGE

>  If you add a custom CMD in the Dockerfile, be sure to include -g daemon off; in the CMD in order for nginx to stay in the foreground, so that Docker can track the process properly (otherwise your container will stop immediately after starting)!

File: docker-run.sh [hunk 2]
```diff
@@ -2,14 +2,9 @@
...
 
-BASE_URL=${BASE_URL:-/}
 NGINX_ROOT=/usr/share/nginx/html
 INDEX_FILE=$NGINX_ROOT/index.html
-NGINX_CONF=/etc/nginx/nginx.conf
-
-if [[ "${BASE_URL}" != "/" ]]; then
-  sed -i "s|location / {|location $BASE_URL {|g" $NGINX_CONF
-fi
+NGINX_CONF=/etc/nginx/conf.d/default.conf
```
Comment (char0n) [inline docker-run.sh:7]: TECHNICAL NOTE EXPLAINING THE CHANGE

image

File: Dockerfile [hunk 3]
```diff
@@ -2,20 +2,16 @@ FROM nginx:1.23.3-alpine
...
 
-
-RUN chmod +x /usr/share/nginx/docker-run.sh && \
-    chmod -R a+rw /usr/share/nginx && \
-    chmod -R a+rw /etc/nginx
+RUN chmod +x /docker-entrypoint.d/91-docker-run.sh
 
 EXPOSE 8080
-
-CMD ["sh", "/usr/share/nginx/docker-run.sh"]
```
Comment (char0n) [inline Dockerfile:21]: TECHNICAL NOT EXPLAINING THE CHANGE

> No need for `CMD` as `/docker-entrypoint.d/91-docker-run.sh` is used now

File: Dockerfile [hunk 4]
```diff
@@ -2,20 +2,16 @@ FROM nginx:1.23.3-alpine
 
 LABEL maintainer="fehguy"
 
-ENV BASE_URL ""
+ENV BASE_URL "/" \
```
[the remark this answers is not in the document]
Comment (char0n) [reply Dockerfile:5]: Can we change the `NGINX_PORT` TO `PORT`? (Just for consistency with SwaggerUI docker image)

File: Dockerfile [hunk 5]
```diff
@@ -2,20 +2,16 @@ FROM nginx:1.23.3-alpine
...
-COPY ./docker-run.sh /usr/share/nginx/
+COPY ./docker-run.sh /docker-entrypoint.d/91-docker-run.sh
 
-
-RUN chmod +x /usr/share/nginx/docker-run.sh && \
-    chmod -R a+rw /usr/share/nginx && \
-    chmod -R a+rw /etc/nginx
+RUN chmod +x /docker-entrypoint.d/91-docker-run.sh
 
 EXPOSE 8080
```
Comment (char0n) [inline Dockerfile:17]: If we allow to change the port via `PORT` env var, how does it relate to `EXPOSE` directive?

File: nginx.conf [hunk 6]
```diff
@@ -1,17 +1,3 @@
-worker_processes      1;
-
```
Comment (char0n) [inline nginx.conf:2]: Are these changes even valid?  Resulting config is not wrapped in `http` anymore.

Can you corroborate why these changes have been made?

**Review by char0n — CHANGES_REQUESTED** [review]
Thanks for the PR!

I left bunch of review comments.

I've tried to run the image locally with:

```sh
$  docker build -t swagger-editor .
$ docker run -d -p 80:8080 swagger-editor
```
But with no luck.

```
CONTAINER ID   IMAGE                           COMMAND                  CREATED         STATUS                      PORTS     NAMES
b193c415677e   swagger-editor                  "/docker-entrypoint.…"   4 seconds ago   Exited (1) 3 seconds ago              angry_mestorf
```

File: Dockerfile (hunk 4 above)
[the remark this answers is not in the document]
Comment (GAS85) [reply Dockerfile:5]: > According to the [docker docs](https://docs.docker.com/engine/reference/builder/#env) this should be defined as following:
> 
> ```
> ENV BASE_URL="/" \
>     NGINX_PORT="8080"
> ```
> 
> Can we change this env defs so that it's aligned with official docker docs?

But this is exactly how it is defined in line 5 and 6. Or am I missing something?

[the remark this answers is not in the document]
Comment (char0n) [reply Dockerfile:5]:
```
ENV BASE_URL "/" \
    NGINX_PORT "8080"
```
vs 
```
ENV BASE_URL="/" \
    NGINX_PORT="8080"
```

File: Dockerfile (hunk 5 above)
Comment (GAS85) [reply Dockerfile:17]: According to the [docker documentation](https://docs.docker.com/engine/reference/builder/#expose) and personal experience it does not really matter: "The EXPOSE instruction **does not actually publish** the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the `-p` flag on `docker run` to publish and map one or more ports, or the `-P` flag to publish all exposed ports and map them to high-order ports."

Comment (char0n) [reply Dockerfile:17]: Sounds good.

File: Dockerfile (hunk 3 above)
Comment (GAS85) [reply Dockerfile:21]: This is the trick. Nginx image will [execute all scripts](https://github.com/nginxinc/docker-nginx/blob/master/mainline/alpine-slim/docker-entrypoint.sh#L16) from the `/docker-entrypoint.d` folder and [`/docker-entrypoint.d/91-docker-run.sh`](https://github.com/swagger-api/swagger-editor/pull/3958/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R13) too (instead of the old location `/usr/share/nginx/docker-run.sh`) before to run whatever provided in `CMD`.
After that default [CMD of nginx container](https://github.com/nginxinc/docker-nginx/blob/master/mainline/alpine-slim/Dockerfile#L121) will be applied that basically runs the nginx in foreground.
So we do not need to rewrite this part, but use default CMD instead, because our script was executed in line 9. And nginx afterwads up line 10:

```shell
# docker logs swagger-editor 2>&1 | nl -w 1
1       /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
2       /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
3       /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
4       10-listen-on-ipv6-by-default.sh: info: Getting the checksum of /etc/nginx/conf.d/default.conf
5       10-listen-on-ipv6-by-default.sh: info: Enabled listen on IPv6 in /etc/nginx/conf.d/default.conf
6       /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
7       20-envsubst-on-templates.sh: Running envsubst on /etc/nginx/templates/default.conf.template to /etc/nginx/conf.d/default.conf
8       /docker-entrypoint.sh: Launching /docker-entrypoint.d/30-tune-worker-processes.sh
9       /docker-entrypoint.sh: Launching /docker-entrypoint.d/91-docker-run.sh
10      2023/03/22 10:18:20 [notice] 35#35: using the "epoll" event method
11      2023/03/22 10:18:20 [notice] 35#35: nginx/1.23.3
12      2023/03/22 10:18:20 [notice] 35#35: built by gcc 12.2.1 20220924 (Alpine 12.2.1_git20220924-r4) 
13      2023/03/22 10:18:20 [notice] 35#35: OS: Linux 5.15.0-67-generic
14      2023/03/22 10:18:20 [notice] 35#35: getrlimit(RLIMIT_NOFILE): 1048576:1048576
15      2023/03/22 10:18:20 [notice] 35#35: start worker processes
16      2023/03/22 10:18:20 [notice] 35#35: start worker process 36
17      2023/03/22 10:18:20 [notice] 35#35: start worker process 37
18      2023/03/22 10:18:20 [notice] 35#35: start worker process 38
19      2023/03/22 10:18:20 [notice] 35#35: start worker process 39
```

This is also the reason, why we do not need to start nginx by our own and I [removed this line](https://github.com/swagger-api/swagger-editor/pull/3958/files#diff-360c358e5fc4de52699ff11376043b7c67ebccc164c8dd9bc2b025782d681279L67)

File: docker-run.sh (hunk 2 above)
Comment (GAS85) [reply docker-run.sh:7]: - We place our configuration into template folder: https://github.com/swagger-api/swagger-editor/pull/3958/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R8
- It will be [modified by the nginx script](https://github.com/nginxinc/docker-nginx/blob/master/mainline/alpine-slim/20-envsubst-on-templates.sh) and moved to the `/etc/nginx/conf.d/<file name>` in our case`/etc/nginx/conf.d/default.conf`, here is the [docu](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119) --> Thats why path was changed.

Comment (char0n) [reply docker-run.sh:7]: Yep, this comment was for reference purposes only. No action on your side was needed.

Comment (char0n) [reply docker-run.sh:7]: Yep, this comment was for reference purposes only. No action on your side was needed.

File: Dockerfile (hunk 3 above)
Comment (char0n) [reply Dockerfile:21]: Yep, this comment was for reference purposes only. No action on your side was needed.

File: docker-run.sh (hunk 1 above)
Comment (char0n) [reply docker-run.sh:67]: Yep, this comment was for reference purposes only. No action on your side was needed.

File: nginx.conf (hunk 6 above)
Comment (GAS85) [reply nginx.conf:2]: - We do place our configuration into template folder: https://github.com/swagger-api/swagger-editor/pull/3958/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R8
- It will be [modified by the nginx script](https://github.com/nginxinc/docker-nginx/blob/master/mainline/alpine-slim/20-envsubst-on-templates.sh) and moved to the `/etc/nginx/conf.d/<file name>` in our case`/etc/nginx/conf.d/default.conf`, here is the [docu](https://github.com/docker-library/docs/tree/master/nginx#using-environment-variables-in-nginx-configuration-new-in-119).
- Nginx by start will apply configuration from the `/etc/nginx/nginx.conf` that is [including at the end]() whatever was provided in a `/etc/nginx/conf.d` folder via `include /etc/nginx/conf.d/*.conf;`:
```nginx
user  nginx;
worker_processes  auto;

error_log  /var/log/nginx/error.log notice;
pid        /var/run/nginx.pid;


events {
    worker_connections  1024;
}


http {
    include       /etc/nginx/mime.types;
    default_type  application/octet-stream;

    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                      '$status $body_bytes_sent "$http_referer" '
                      '"$http_user_agent" "$http_x_forwarded_for"';

    access_log  /var/log/nginx/access.log  main;

    sendfile        on;
    #tcp_nopush     on;

    keepalive_timeout  65;

    #gzip  on;

    include /etc/nginx/conf.d/*.conf;
}
```
- as include is already on `http` level, we have no need to configure working processes, etc. by our own anymore, but **relay on only partly Service configuration**.
- Whole active configuration will looks like below (New Active Configuration), where Default config applied in lines from 1 till 21 and our own configuration add up line 115. 
You can compare it with a Original Swagger container active configuration.

<details>

<summary>New Active Configuration</summary>

```shell
# docker exec swagger-editro nginx -T | nl -w 1
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
1       # configuration file /etc/nginx/nginx.conf:
  
2       user  nginx;
3       worker_processes  auto;
  
4       error_log  /var/log/nginx/error.log notice;
5       pid        /var/run/nginx.pid;
  
  
6       events {
7           worker_connections  1024;
8       }
  
  
9       http {
10          include       /etc/nginx/mime.types;
11          default_type  application/octet-stream;
  
12          log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
13                            '$status $body_bytes_sent "$http_referer" '
14                            '"$http_user_agent" "$http_x_forwarded_for"';
  
15          access_log  /var/log/nginx/access.log  main;
  
16          sendfile        on;
17          #tcp_nopush     on;
  
18          keepalive_timeout  65;
  
19          #gzip  on;
  
20          include /etc/nginx/conf.d/*.conf;
21      }
  
22      # configuration file /etc/nginx/mime.types:
  
23      types {
24          text/html                                        html htm shtml;
25          text/css                                         css;
26          text/xml                                         xml;
27          image/gif                                        gif;
28          image/jpeg                                       jpeg jpg;
29          application/javascript                           js;
30          application/atom+xml                             atom;
31          application/rss+xml                              rss;
  
32          text/mathml                                      mml;
33          text/plain                                       txt;
34          text/vnd.sun.j2me.app-descriptor                 jad;
35          text/vnd.wap.wml                                 wml;
36          text/x-component                                 htc;
  
37          image/avif                                       avif;
38          image/png                                        png;
39          image/svg+xml                                    svg svgz;
40          image/tiff                                       tif tiff;
41          image/vnd.wap.wbmp                               wbmp;
42          image/webp                                       webp;
43          image/x-icon                                     ico;
44          image/x-jng                                      jng;
45          image/x-ms-bmp                                   bmp;
  
46          font/woff                                        woff;
47          font/woff2                                       woff2;
  
48          application/java-archive                         jar war ear;
49          application/json                                 json;
50          application/mac-binhex40                         hqx;
51          application/msword                               doc;
52          application/pdf                                  pdf;
53          application/postscript                           ps eps ai;
54          application/rtf                                  rtf;
55          application/vnd.apple.mpegurl                    m3u8;
56          application/vnd.google-earth.kml+xml             kml;
57          application/vnd.google-earth.kmz                 kmz;
58          application/vnd.ms-excel                         xls;
59          application/vnd.ms-fontobject                    eot;
60          application/vnd.ms-powerpoint                    ppt;
61          application/vnd.oasis.opendocument.graphics      odg;
62          application/vnd.oasis.opendocument.presentation  odp;
63          application/vnd.oasis.opendocument.spreadsheet   ods;
64          application/vnd.oasis.opendocument.text          odt;
65          application/vnd.openxmlformats-officedocument.presentationml.presentation
66                                                           pptx;
67          application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
68                                                           xlsx;
69          application/vnd.openxmlformats-officedocument.wordprocessingml.document
70                                                           docx;
71          application/vnd.wap.wmlc                         wmlc;
72          application/wasm                                 wasm;
73          application/x-7z-compressed                      7z;
74          application/x-cocoa                              cco;
75          application/x-java-archive-diff                  jardiff;
76          application/x-java-jnlp-file                     jnlp;
77          application/x-makeself                           run;
78          application/x-perl                               pl pm;
79          application/x-pilot                              prc pdb;
80          application/x-rar-compressed                     rar;
81          application/x-redhat-package-manager             rpm;
82          application/x-sea                                sea;
83          application/x-shockwave-flash                    swf;
84          application/x-stuffit                            sit;
85          application/x-tcl                                tcl tk;
86          application/x-x509-ca-cert                       der pem crt;
87          application/x-xpinstall                          xpi;
88          application/xhtml+xml                            xhtml;
89          application/xspf+xml                             xspf;
90          application/zip                                  zip;
  
91          application/octet-stream                         bin exe dll;
92          application/octet-stream                         deb;
93          application/octet-stream                         dmg;
94          application/octet-stream                         iso img;
95          application/octet-stream                         msi msp msm;
  
96          audio/midi                                       mid midi kar;
97          audio/mpeg                                       mp3;
98          audio/ogg                                        ogg;
99          audio/x-m4a                                      m4a;
100         audio/x-realaudio                                ra;
  
101         video/3gpp                                       3gpp 3gp;
102         video/mp2t                                       ts;
103         video/mp4                                        mp4;
104         video/mpeg                                       mpeg mpg;
105         video/quicktime                                  mov;
106         video/webm                                       webm;
107         video/x-flv                                      flv;
108         video/x-m4v                                      m4v;
109         video/x-mng                                      mng;
110         video/x-ms-asf                                   asx asf;
111         video/x-ms-wmv                                   wmv;
112         video/x-msvideo                                  avi;
113     }
  
114     # configuration file /etc/nginx/conf.d/default.conf:
115       gzip on;
116       gzip_static on;
117       gzip_disable "msie6";
  
118       gzip_vary on;
119       gzip_types text/plain text/css application/javascript;
120       
121       add_header X-Frame-Options deny;
122       server_tokens off; # Hide Nginx version
  
123       server {
124         listen            8080;
125         server_name       localhost;
  
126         location / {
127           absolute_redirect off;
128           alias           /usr/share/nginx/html/;
129           index           index.html index.htm;
  
130           location ~* \.(?:json|yml|yaml)$ {
131             #SWAGGER_ROOT
132             expires -1;
133           }
134         }
135       }
```
</details>

<details>

<summary>Original Swagger container active configuration</summary>

```bash
# docker exec swagger nginx -T | nl -w 2
nginx: the configuration file /etc/nginx/nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is successful
 1      # configuration file /etc/nginx/nginx.conf:
 2      worker_processes      1;
   
 3      events {
 4        worker_connections  1024;
 5      }
   
 6      http {
 7        include             mime.types;
 8        default_type        application/octet-stream;
   
 9        sendfile on;
   
10        keepalive_timeout   65;
   
11        gzip on;
12        gzip_static on;
13        gzip_disable "msie6";
   
14        gzip_vary on;
15        gzip_types text/plain text/css application/javascript;
16        
17        add_header X-Frame-Options deny;
18        server_tokens off; # Hide Nginx version
   
19        server {
20          listen            8080;
21          server_name       localhost;
   
22          location / {
23            absolute_redirect off;
24            alias           /usr/share/nginx/html/;
25            index           index.html index.htm;
   
26            location ~* \.(?:json|yml|yaml)$ {
27              #SWAGGER_ROOT
28              expires -1;
29            }
30          }
31        }
32      }
   
33      # configuration file /etc/nginx/mime.types:
   
34      types {
35          text/html                                        html htm shtml;
36          text/css                                         css;
37          text/xml                                         xml;
38          image/gif                                        gif;
39          image/jpeg                                       jpeg jpg;
40          application/javascript                           js;
41          application/atom+xml                             atom;
42          application/rss+xml                              rss;
   
43          text/mathml                                      mml;
44          text/plain                                       txt;
45          text/vnd.sun.j2me.app-descriptor                 jad;
46          text/vnd.wap.wml                                 wml;
47          text/x-component                                 htc;
   
48          image/avif                                       avif;
49          image/png                                        png;
50          image/svg+xml                                    svg svgz;
51          image/tiff                                       tif tiff;
52          image/vnd.wap.wbmp                               wbmp;
53          image/webp                                       webp;
54          image/x-icon                                     ico;
55          image/x-jng                                      jng;
56          image/x-ms-bmp                                   bmp;
   
57          font/woff                                        woff;
58          font/woff2                                       woff2;
   
59          application/java-archive                         jar war ear;
60          application/json                                 json;
61          application/mac-binhex40                         hqx;
62          application/msword                               doc;
63          application/pdf                                  pdf;
64          application/postscript                           ps eps ai;
65          application/rtf                                  rtf;
66          application/vnd.apple.mpegurl                    m3u8;
67          application/vnd.google-earth.kml+xml             kml;
68          application/vnd.google-earth.kmz                 kmz;
69          application/vnd.ms-excel                         xls;
70          application/vnd.ms-fontobject                    eot;
71          application/vnd.ms-powerpoint                    ppt;
72          application/vnd.oasis.opendocument.graphics      odg;
73          application/vnd.oasis.opendocument.presentation  odp;
74          application/vnd.oasis.opendocument.spreadsheet   ods;
75          application/vnd.oasis.opendocument.text          odt;
76          application/vnd.openxmlformats-officedocument.presentationml.presentation
77                                                           pptx;
78          application/vnd.openxmlformats-officedocument.spreadsheetml.sheet
79                                                           xlsx;
80          application/vnd.openxmlformats-officedocument.wordprocessingml.document
81                                                           docx;
82          application/vnd.wap.wmlc                         wmlc;
83          application/wasm                                 wasm;
84          application/x-7z-compressed                      7z;
85          application/x-cocoa                              cco;
86          application/x-java-archive-diff                  jardiff;
87          application/x-java-jnlp-file                     jnlp;
88          application/x-makeself                           run;
89          application/x-perl                               pl pm;
90          application/x-pilot                              prc pdb;
91          application/x-rar-compressed                     rar;
92          application/x-redhat-package-manager             rpm;
93          application/x-sea                                sea;
94          application/x-shockwave-flash                    swf;
95          application/x-stuffit                            sit;
96          application/x-tcl                                tcl tk;
97          application/x-x509-ca-cert                       der pem crt;
98          application/x-xpinstall                          xpi;
99          application/xhtml+xml                            xhtml;
100         application/xspf+xml                             xspf;
101         application/zip                                  zip;
   
102         application/octet-stream                         bin exe dll;
103         application/octet-stream                         deb;
104         application/octet-stream                         dmg;
105         application/octet-stream                         iso img;
106         application/octet-stream                         msi msp msm;
   
107         audio/midi                                       mid midi kar;
108         audio/mpeg                                       mp3;
109         audio/ogg                                        ogg;
110         audio/x-m4a                                      m4a;
111         audio/x-realaudio                                ra;
   
112         video/3gpp                                       3gpp 3gp;
113         video/mp2t                                       ts;
114         video/mp4                                        mp4;
115         video/mpeg                                       mpeg mpg;
116         video/quicktime                                  mov;
117         video/webm                                       webm;
118         video/x-flv                                      flv;
119         video/x-m4v                                      m4v;
120         video/x-mng                                      mng;
121         video/x-ms-asf                                   asx asf;
122         video/x-ms-wmv                                   wmv;
123         video/x-msvideo                                  avi;
124     }
```

File: Dockerfile (hunk 4 above)
[the remark this answers is not in the document]
Comment (GAS85) [reply Dockerfile]: Thanks, I didn't noticed it.
Now `ENV`s and `PORT` are applied.

**Comment by char0n** [comment]
> Just stupid question, did you run npm build before?

@GAS85 nope, build fragments are checked-out inside the codebase, so this step shouldn't be needed.

**Review by char0n — APPROVED** [review]
Looks good to me. Great work!

I was able to test locally all the changes and they check out.

**Comment by GAS85** [comment]
Thanks! I Appreciate it!
